app/api/events.py

```diff
@@ -118,8 +118,12 @@ def before_post(self, args, kwargs, data=None):
         user = User.query.filter_by(id=kwargs['user_id']).first()
         modules = Module.query.first()
         if data.get('is_ticketing_enabled', False) and not modules.ticket_include:
-            raise ForbiddenException({'source': '/data/attributes/is-ticketing-enabled'},
-                                     "Ticketing is not enabled in the system")
+            if user.is_verified:
```

Comment:
These conditions are in wrong place. They have nothing to do with tickets module. Add them as separate conditions